bayesian_topological_localisation/DistributionStamped likelihood
string identity